homeassistant/components/notify/facebook.py

```diff
@@ -55,20 +58,28 @@ def send_message(self, message="", **kwargs):
             _LOGGER.error("At least 1 target is required")
             return
 
-        for target in targets:
-            # If the target starts with a "+", we suppose it's a phone number,
-            # otherwise it's a user id.
-            if target.startswith('+'):
-                recipient = {"phone_number": target}
-            else:
-                recipient = {"id": target}
-
-            body = {
-                "recipient": recipient,
-                "message": body_message
+        # broadcast message
+        if targets[0].startswith('BROADCAST'):
+
+            brdcast_create_body = {"messages": [body_message]}
+
+            _LOGGER.debug("Broadcast body %s : ", brdcast_create_body)
+
+            resp = requests.post(CR_BRCST_MSG_URL,
+                                 data=json.dumps(brdcast_create_body),
+                                 params=payload,
+                                 headers={CONTENT_TYPE: CONTENT_TYPE_JSON},
+                                 timeout=10)
+            _LOGGER.info("FB Messager broadcast id %s : ", resp.json())
```

Comment:
Make this `debug` or remove it as it doesn't contain details one would need during a normal usage.